bin/scripts/lib/fonts.json

```diff
@@ -615,6 +615,17 @@
       "caskName": "victor-mono",
       "repoRelease": true,
       "description": "Clean, crisp and narrow, with a large x-height and clear punctuation"
+    },
+    {
+      "unpatchedName": "Geist Mono",
+      "RFN": false,
+      "patchedName": "GeistMono",
+      "folderName": "GeistMono",
+      "imagePreviewFont": "GeistMono Nerd Font",
+      "linkPreviewFont": "geist",
+      "caskName": "geist-mono",
+      "repoRelease": false,
+      "description": "Created by Vercel in collaboration with Basement Studio"
```

Comment:
This is a bit non-descriptive, usually here comes some info how the font distinguishes against all other fonts, what makes it great. Well, if there is something ;-) Maybe the description is ok, I do not know.